project/src/main/java/com/google/sps/servlets/PostEventServlet.java

```diff
@@ -32,19 +32,49 @@
 import javax.servlet.http.HttpServlet;
 import javax.servlet.http.HttpServletRequest;
 import javax.servlet.http.HttpServletResponse;
+import javax.servlet.http.HttpSession;
 import com.google.sps.classes.User;
 import com.google.appengine.api.users.UserService;
 import com.google.appengine.api.users.UserServiceFactory;
 import java.text.SimpleDateFormat;
 import java.text.ParseException;
+import com.google.appengine.api.blobstore.BlobKey;
+import com.google.appengine.api.blobstore.BlobstoreService;
+import com.google.appengine.api.blobstore.BlobstoreServiceFactory;
+import com.google.appengine.api.blobstore.BlobInfo;
+import com.google.appengine.api.blobstore.BlobInfoFactory;
+import java.util.Map;
+import com.google.gson.Gson;
  
 /** Servlet that posts an event*/
 @WebServlet("/post-event")
 public class PostEventServlet extends HttpServlet {
  
   @Override
-  public void doPost(HttpServletRequest request, HttpServletResponse response) throws IOException {
+  public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
+    
+    List<Boolean> log = new ArrayList<>();
+ 
+    HttpSession session = request.getSession(false); 
+    String name = (String) session.getAttribute("person"); 
+ 
+    boolean found = false;
+    
+    if (!name.equals("null")) {
+      found = true;
```

Comment:
You could initialize found to true and then do

```
if (name.equals("null")) {
    found = false;
}
```
to avoid the ! on the equals, but up to you.

Reply:
I am still not implementing this method correctly, just waiting on your admin branch to get merged!